packages/frontend/components/composed/entity/EntityCitationMetaTags/EntityCitationMetaTags.tsx

```diff
@@ -0,0 +1,191 @@
+import React from "react";
+import Head from "next/head";
+
+import { graphql } from "react-relay";
+import { useMaybeFragment } from "@wdp/lib/api/hooks";
+import { EntityCitationMetaTagsFragment$key } from "@/relay/EntityCitationMetaTagsFragment.graphql";
+import { getPrecisionCitationDateDisplay } from "helpers";
+import { getContributorDisplayName } from "components/composed/contributor/ContributorName/ContributorName";
+
+/**
+ * These meta tags should only be included on the home page of an item.
+ * For citation guidelines, see https://scholar.google.com/intl/en/scholar/inclusion.html#indexing
+ */
+export default function EntityCitationMetaTags({ data }: Props) {
+  const entity = useMaybeFragment(fragment, data);
+
+  const identifier = entity?.schemaDefinition?.identifier;
+
+  // If the entity is null or the schema identifier doesn't match, return null.
+  if (
+    !entity ||
+    !identifier ||
+    !["article", "journal_article", "dissertation", "paper"].includes(
+      identifier
+    )
+  )
+    return null;
+
+  const volume = entity.volumeNumber?.value || entity.volume?.number?.value;
+
+  const issue = entity.issueNumber?.value || entity.issue?.number?.value;
+
+  const startPage = entity?.startPage?.value;
+
+  const endPage = entity?.endPage?.value;
+
+  const pdf = entity.pdf?.asset?.downloadUrl;
+
+  const published = getPrecisionCitationDateDisplay(
+    entity.published?.precision,
+    entity.published?.value
+  );
+
+  const journal = entity.journal?.title;
+
+  const institution = entity.institution?.value || entity.community?.title;
+
+  const authors =
+    entity.contributions?.nodes?.filter(
+      ({ role }: { role: string | null }) => role === "author"
+    ) || [];
+
+  return (
+    <Head>
+      <meta name="citation_title" content={entity.title} />
+      {authors.map(({ contributor }, i) => (
+        <meta
+          key={i}
+          name="citation_author"
+          content={getContributorDisplayName(contributor, true)}
+        />
+      ))}
+      {published && (
+        <meta name="citation_publication_date" content={published} />
+      )}
+      {journal && <meta name="citation_journal_title" content={journal} />}
+      {volume && <meta name="citation_volume" content={volume} />}
+      {issue && issue !== "0" && <meta name="citation_issue" content={issue} />}
+      {startPage && (
+        <meta name="citation_firstpage" content={String(startPage)} />
+      )}
+      {endPage && <meta name="citation_lastpage" content={String(endPage)} />}
+      {pdf && <meta name="citation_pdf_url" content={pdf} />}
+      {institution && identifier === "dissertation" && (
+        <meta name="citation_dissertation_institution" content={institution} />
+      )}
+      {/* citation_technical_report_institution */}
+      {/* citation_technical_report_number */}
+    </Head>
```

Comment:
@dananjohnson , Zach mentioned that you had to add Google Scholar meta tags to Manifold. Can you give this a quick look to see if I'm missing anything? We have articles, dissertations, journals and papers in NGLP. The one thing that wasn't clear to me is if this info needs to be included on a journal homepage. Do you know if these tags need to be included on the journal itself? Or just the papers inside it?